packages/manager/modules/vps/src/upscale/configuration-price/configuration-price.controller.js

```diff
@@ -0,0 +1,20 @@
+import isString from 'lodash/isString';
+import PriceService from '../price/price.service';
+
+export default class {
+  $onInit() {
+    this.validateLanguage();
+    PriceService.validatePrice(this.price);
+
+    // the variables used in the view
+    this.v = {
```

Comment:
can you rename v to something meaningful?

Reply:
It's the conventional name for the view bindings